### ISSUE TITLE
Can't log in, Invalid Credentials but Password and Mail are Correct

### ISSUE THREAD

**Issue by nrnn343** [MiniGlome/Archive.org-Downloader#123]
This great tool no longer works. Tried with new accounts made post the ddos attack, same result

**Comment by daniel-wojahn** [comment]
yeah, same here, unfortunately

**Comment by MiniGlome** [comment]
Hello, did you pull the last commit (https://github.com/MiniGlome/Archive.org-Downloader/commit/d05c84192095ab89f2d0cdf1c7dbefc5cfc8747d)? It should have been fixed a few days ago.

**Comment by nrnn343** [comment]
Yes I did pull it, same Invalid Credentials message. Maybe they changed something again?

**Comment by nrnn343** [comment]
Works again now at time of this message so it's was on their end it seems. I did change my password to something with no symbols in it.

**Comment by daniel-wojahn** [comment]
Yes, the change of password to something without symbols did the trick for me, too! Cheers @nrnn343

**Comment by stuffed18** [comment]
Change the title back to what it was so people with the same issue can find the thread, and then close the thread. There is an option to close it when u reply.
